### PR TITLE
React1 week1

### PR DESCRIPTION
**Pull request by bhumikama**
The following tasks were implemented:

Added a new element to the Footer and imported the Footer component into layout.js.
Added three new components in separate files:
OurValues.js: Displays company values .
OurCrew.js: Displays team members with their photos and descriptions.
OurPartners.js: Displays partner logos

Screenshots of /about_us page
Screenshot 2024-10-08 at 00 04 02
Screenshot 2024-10-08 at 01 13 35
Screenshot 2024-10-08 at 00 06 39
Screenshot 2024-10-08 at 00 08 50

### CODE REVIEW

File: app/about_us/OurCrew.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  return (
+    <>
+      <h3 className={styles.headerThree}>
+        Our crew is the heart and soul of Galactica. We are a diverse team of
+        seasoned space explorers, engineers, and visionaries who are united by a
+        common goal: to make space travel accessible and exciting for all.
+      </h3>
+      <div className={styles.crewContainer}>
+        {crewFields.map((crew, index) => (
+          <div className="card" key={index}>
```
Comment (vdaguenet) [inline app/about_us/OurCrew.js]: ~~This card could be extracted as a component, like `<CrewMemberCard />`~~ I just realised that it is the topic of next week's lesson, so disregard this comment.

File: app/about_us/OurPartners.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    logoUrl: "./business_partners/sodexo-logo.png",
+  },
+];
+export default function OurPartners() {
+  return (
+    <>
+      <h3 className={styles.heading}>We collaborate with some of the most respected names in the space and technology industries to make every journey extraordinary.</h3>
+      <div className={styles.logoContainer}>
+        {
+        partners.map((partner) => (
```
Comment (vdaguenet) [inline app/about_us/OurPartners.js:42]: Same comment as for `<OurCrew />`

File: app/about_us/OurValues.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      "We believe in the power of collective exploration. Our journeys are not just about reaching new destinations; they are about building a community of space enthusiasts who share a passion for the stars.",
+  },
+];
+
+export default function OurValues() {
+  return (
+    <>
+      <div className={styles.valueContainer}>
+        {valueFields.map((value,index) => (
+          <div className={styles.value} key={index}>
```
Comment (vdaguenet) [inline app/about_us/OurValues.js:35]: Same comment as for `<OurCrew />`

File: app/about_us/crew.module.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  padding: 5px 0;
+  font-size: 15px;
+  font-weight: 100;
+  line-height: 1.2;
+  text-align: justify;
+  word-spacing: normal;
+}
+
+@media (max-width: 768px){
+    .crewContainer{
```
Comment (vdaguenet) [inline app/about_us/crew.module.css]: Be careful to not mix 2-spaces and 4-spaces indentation

**Review by vdaguenet — CHANGES_REQUESTED** [review]
Looping over an array for displaying the different cards is an excellent idea 👍🏻 

Edit 1:
~~Also be careful to follow the coding convention of the project: file names are lowercase (`page.js`, `layout.js`)~~ Nevermind, the project uses both.

Edit 2:
~~You should go further into the React way of coding by adding more components.~~ I just realised that it is the topic of next week's lesson, so disregard this comment.

**Review by vdaguenet — APPROVED** [review]

File: app/about_us/OurCrew.js (hunk 1 above)
Comment (bhumikama) [reply app/about_us/OurCrew.js]: Thank you for the review.I will try implementing here as well.

File: app/about_us/crew.module.css (hunk 4 above)
Comment (bhumikama) [reply app/about_us/crew.module.css]: Done. I will take care of these from now on.

**Comment by bhumikama** [comment]
Thanks @vdaguenet for your inputs. I have included the suggested changes. Please have another look.

**Review by vdaguenet — APPROVED** [review]
Excellent

**Pull request merged**